[2024-04-14T20:39:29.191-0400] {processor.py:161} INFO - Started process (PID=38050) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T20:39:29.191-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T20:39:29.192-0400] {logging_mixin.py:188} INFO - [2024-04-14T20:39:29.191-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T20:39:29.230-0400] {logging_mixin.py:188} INFO - [2024-04-14T20:39:29.229-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T20:39:29.230-0400] {logging_mixin.py:188} INFO - [2024-04-14T20:39:29.230-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T20:39:29.230-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T20:39:29.255-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.065 seconds
[2024-04-14T21:46:23.499-0400] {processor.py:161} INFO - Started process (PID=38202) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T21:46:23.500-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T21:46:23.500-0400] {logging_mixin.py:188} INFO - [2024-04-14T21:46:23.500-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T21:46:23.539-0400] {logging_mixin.py:188} INFO - [2024-04-14T21:46:23.539-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T21:46:23.539-0400] {logging_mixin.py:188} INFO - [2024-04-14T21:46:23.539-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T21:46:23.539-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T21:46:23.565-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.066 seconds
[2024-04-14T21:48:24.035-0400] {processor.py:161} INFO - Started process (PID=38313) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T21:48:24.036-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T21:48:24.036-0400] {logging_mixin.py:188} INFO - [2024-04-14T21:48:24.036-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T21:48:24.077-0400] {logging_mixin.py:188} INFO - [2024-04-14T21:48:24.076-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T21:48:24.077-0400] {logging_mixin.py:188} INFO - [2024-04-14T21:48:24.077-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T21:48:24.077-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T21:48:24.102-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.068 seconds
[2024-04-14T21:50:20.748-0400] {processor.py:161} INFO - Started process (PID=38409) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T21:50:20.749-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T21:50:20.750-0400] {logging_mixin.py:188} INFO - [2024-04-14T21:50:20.750-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T21:50:20.787-0400] {logging_mixin.py:188} INFO - [2024-04-14T21:50:20.787-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T21:50:20.787-0400] {logging_mixin.py:188} INFO - [2024-04-14T21:50:20.787-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T21:50:20.787-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T21:50:20.812-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.065 seconds
[2024-04-14T21:52:17.635-0400] {processor.py:161} INFO - Started process (PID=38510) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T21:52:17.635-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T21:52:17.635-0400] {logging_mixin.py:188} INFO - [2024-04-14T21:52:17.635-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T21:52:17.674-0400] {logging_mixin.py:188} INFO - [2024-04-14T21:52:17.674-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T21:52:17.674-0400] {logging_mixin.py:188} INFO - [2024-04-14T21:52:17.674-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T21:52:17.675-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T21:52:17.700-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.066 seconds
[2024-04-14T21:54:07.960-0400] {processor.py:161} INFO - Started process (PID=38595) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T21:54:07.961-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T21:54:07.961-0400] {logging_mixin.py:188} INFO - [2024-04-14T21:54:07.961-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T21:54:08.002-0400] {logging_mixin.py:188} INFO - [2024-04-14T21:54:08.002-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T21:54:08.002-0400] {logging_mixin.py:188} INFO - [2024-04-14T21:54:08.002-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T21:54:08.002-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T21:54:08.028-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.068 seconds
[2024-04-14T21:56:09.463-0400] {processor.py:161} INFO - Started process (PID=38687) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T21:56:09.464-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T21:56:09.464-0400] {logging_mixin.py:188} INFO - [2024-04-14T21:56:09.464-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T21:56:09.507-0400] {logging_mixin.py:188} INFO - [2024-04-14T21:56:09.506-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T21:56:09.507-0400] {logging_mixin.py:188} INFO - [2024-04-14T21:56:09.507-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T21:56:09.507-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T21:56:09.535-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.073 seconds
[2024-04-14T21:58:10.945-0400] {processor.py:161} INFO - Started process (PID=38786) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T21:58:10.946-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T21:58:10.946-0400] {logging_mixin.py:188} INFO - [2024-04-14T21:58:10.946-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T21:58:10.985-0400] {logging_mixin.py:188} INFO - [2024-04-14T21:58:10.985-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T21:58:10.986-0400] {logging_mixin.py:188} INFO - [2024-04-14T21:58:10.986-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T21:58:10.986-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T21:58:11.011-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.067 seconds
[2024-04-14T22:00:08.366-0400] {processor.py:161} INFO - Started process (PID=38894) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:00:08.366-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T22:00:08.367-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:00:08.367-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:00:08.406-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:00:08.406-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T22:00:08.407-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:00:08.407-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T22:00:08.407-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:00:08.432-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.068 seconds
[2024-04-14T22:02:02.475-0400] {processor.py:161} INFO - Started process (PID=38993) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:02:02.477-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T22:02:02.477-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:02:02.477-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:02:02.514-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:02:02.514-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T22:02:02.514-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:02:02.514-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T22:02:02.514-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:02:02.540-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.065 seconds
[2024-04-14T22:04:00.626-0400] {processor.py:161} INFO - Started process (PID=39091) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:04:00.627-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T22:04:00.627-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:04:00.627-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:04:00.665-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:04:00.664-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T22:04:00.665-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:04:00.665-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T22:04:00.665-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:04:00.690-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.065 seconds
[2024-04-14T22:05:55.687-0400] {processor.py:161} INFO - Started process (PID=39189) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:05:55.688-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T22:05:55.688-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:05:55.688-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:05:55.728-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:05:55.727-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T22:05:55.728-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:05:55.728-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T22:05:55.728-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:05:55.754-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.068 seconds
[2024-04-14T22:07:51.289-0400] {processor.py:161} INFO - Started process (PID=39285) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:07:51.290-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T22:07:51.291-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:07:51.291-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:07:51.329-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:07:51.328-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T22:07:51.329-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:07:51.329-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T22:07:51.329-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:07:51.354-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.066 seconds
[2024-04-14T22:09:49.426-0400] {processor.py:161} INFO - Started process (PID=39380) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:09:49.427-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T22:09:49.427-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:09:49.427-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:09:49.465-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:09:49.465-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T22:09:49.465-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:09:49.465-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T22:09:49.465-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:09:49.490-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.066 seconds
[2024-04-14T22:11:54.554-0400] {processor.py:161} INFO - Started process (PID=39512) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:11:54.554-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T22:11:54.555-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:11:54.555-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:11:54.593-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:11:54.593-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T22:11:54.594-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:11:54.593-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T22:11:54.594-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:11:54.622-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.069 seconds
[2024-04-14T22:13:53.079-0400] {processor.py:161} INFO - Started process (PID=39643) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:13:53.080-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T22:13:53.080-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:13:53.080-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:13:53.125-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:13:53.125-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T22:13:53.125-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:13:53.125-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T22:13:53.125-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:13:53.153-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.074 seconds
[2024-04-14T22:15:56.452-0400] {processor.py:161} INFO - Started process (PID=39741) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:15:56.453-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T22:15:56.455-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:15:56.455-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:15:56.507-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:15:56.507-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T22:15:56.507-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:15:56.507-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T22:15:56.508-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:15:56.535-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.085 seconds
[2024-04-14T22:17:57.842-0400] {processor.py:161} INFO - Started process (PID=39893) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:17:57.843-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T22:17:57.843-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:17:57.843-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:17:57.887-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:17:57.887-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T22:17:57.887-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:17:57.887-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T22:17:57.887-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:17:57.914-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.073 seconds
[2024-04-14T22:19:56.025-0400] {processor.py:161} INFO - Started process (PID=40008) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:19:56.027-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T22:19:56.027-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:19:56.027-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:19:56.068-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:19:56.068-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T22:19:56.069-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:19:56.069-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T22:19:56.069-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:19:56.095-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.070 seconds
[2024-04-14T22:21:46.702-0400] {processor.py:161} INFO - Started process (PID=40104) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:21:46.703-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T22:21:46.704-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:21:46.704-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:21:46.743-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:21:46.743-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T22:21:46.744-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:21:46.744-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T22:21:46.744-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:21:46.769-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.068 seconds
[2024-04-14T22:23:44.689-0400] {processor.py:161} INFO - Started process (PID=40233) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:23:44.690-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T22:23:44.690-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:23:44.690-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:23:44.742-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:23:44.742-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T22:23:44.742-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:23:44.742-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T22:23:44.742-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:23:44.770-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.083 seconds
[2024-04-14T22:25:42.945-0400] {processor.py:161} INFO - Started process (PID=40330) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:25:42.946-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T22:25:42.946-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:25:42.946-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:25:42.985-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:25:42.984-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T22:25:42.985-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:25:42.985-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T22:25:42.985-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:25:43.011-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.067 seconds
[2024-04-14T22:27:39.181-0400] {processor.py:161} INFO - Started process (PID=40421) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:27:39.182-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T22:27:39.182-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:27:39.182-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:27:39.220-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:27:39.220-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T22:27:39.220-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:27:39.220-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T22:27:39.220-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:27:39.246-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.066 seconds
[2024-04-14T22:29:33.090-0400] {processor.py:161} INFO - Started process (PID=40527) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:29:33.091-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T22:29:33.091-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:29:33.091-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:29:33.129-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:29:33.129-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T22:29:33.129-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:29:33.129-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T22:29:33.129-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:29:33.155-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.066 seconds
[2024-04-14T22:31:33.508-0400] {processor.py:161} INFO - Started process (PID=40631) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:31:33.510-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T22:31:33.510-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:31:33.510-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:31:33.548-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:31:33.548-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T22:31:33.548-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:31:33.548-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T22:31:33.548-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:31:33.574-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.066 seconds
[2024-04-14T22:33:32.644-0400] {processor.py:161} INFO - Started process (PID=40725) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:33:32.645-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T22:33:32.645-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:33:32.645-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:33:32.683-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:33:32.683-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T22:33:32.683-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:33:32.683-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T22:33:32.683-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:33:32.709-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.066 seconds
[2024-04-14T22:35:32.452-0400] {processor.py:161} INFO - Started process (PID=40815) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:35:32.452-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T22:35:32.453-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:35:32.453-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:35:32.491-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:35:32.491-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T22:35:32.491-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:35:32.491-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T22:35:32.492-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:35:32.518-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.068 seconds
[2024-04-14T22:37:31.301-0400] {processor.py:161} INFO - Started process (PID=40904) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:37:31.302-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T22:37:31.302-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:37:31.302-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:37:31.341-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:37:31.341-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T22:37:31.341-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:37:31.341-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T22:37:31.342-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:37:31.367-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.067 seconds
[2024-04-14T22:39:29.120-0400] {processor.py:161} INFO - Started process (PID=41004) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:39:29.121-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T22:39:29.122-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:39:29.121-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:39:29.161-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:39:29.161-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T22:39:29.161-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:39:29.161-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T22:39:29.162-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:39:29.187-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.068 seconds
[2024-04-14T22:41:27.360-0400] {processor.py:161} INFO - Started process (PID=41100) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:41:27.361-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T22:41:27.361-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:41:27.361-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:41:27.400-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:41:27.400-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T22:41:27.400-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:41:27.400-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T22:41:27.400-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:41:27.426-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.067 seconds
[2024-04-14T22:43:28.309-0400] {processor.py:161} INFO - Started process (PID=41193) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:43:28.310-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T22:43:28.311-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:43:28.311-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:43:28.349-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:43:28.349-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T22:43:28.349-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:43:28.349-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T22:43:28.349-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:43:28.375-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.067 seconds
[2024-04-14T22:45:21.918-0400] {processor.py:161} INFO - Started process (PID=41284) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:45:21.919-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T22:45:21.919-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:45:21.919-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:45:21.958-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:45:21.958-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T22:45:21.958-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:45:21.958-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T22:45:21.958-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:45:21.984-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.067 seconds
[2024-04-14T22:47:22.419-0400] {processor.py:161} INFO - Started process (PID=41382) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:47:22.420-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T22:47:22.420-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:47:22.420-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:47:22.461-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:47:22.461-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T22:47:22.461-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:47:22.461-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T22:47:22.461-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:47:22.487-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.069 seconds
[2024-04-14T22:49:17.798-0400] {processor.py:161} INFO - Started process (PID=41477) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:49:17.798-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T22:49:17.798-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:49:17.798-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:49:17.837-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:49:17.837-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T22:49:17.837-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:49:17.837-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T22:49:17.838-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:49:17.863-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.067 seconds
[2024-04-14T22:51:14.651-0400] {processor.py:161} INFO - Started process (PID=41565) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:51:14.654-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T22:51:14.654-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:51:14.654-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:51:14.692-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:51:14.692-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T22:51:14.692-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:51:14.692-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T22:51:14.692-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:51:14.717-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.067 seconds
[2024-04-14T22:53:08.729-0400] {processor.py:161} INFO - Started process (PID=41654) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:53:08.730-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T22:53:08.730-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:53:08.730-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:53:08.768-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:53:08.768-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T22:53:08.769-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:53:08.769-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T22:53:08.769-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:53:08.794-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.066 seconds
[2024-04-14T22:55:05.797-0400] {processor.py:161} INFO - Started process (PID=41752) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:55:05.798-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T22:55:05.798-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:55:05.798-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:55:05.841-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:55:05.841-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T22:55:05.841-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:55:05.841-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T22:55:05.841-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:55:05.867-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.072 seconds
[2024-04-14T22:57:04.984-0400] {processor.py:161} INFO - Started process (PID=41847) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:57:04.985-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T22:57:04.985-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:57:04.985-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:57:05.025-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:57:05.025-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T22:57:05.025-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:57:05.025-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T22:57:05.025-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:57:05.051-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.068 seconds
[2024-04-14T22:58:56.121-0400] {processor.py:161} INFO - Started process (PID=41974) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:58:56.122-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T22:58:56.122-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:58:56.122-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:58:56.161-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:58:56.161-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T22:58:56.161-0400] {logging_mixin.py:188} INFO - [2024-04-14T22:58:56.161-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T22:58:56.161-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T22:58:56.187-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.067 seconds
[2024-04-14T23:01:00.012-0400] {processor.py:161} INFO - Started process (PID=42123) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:01:00.013-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T23:01:00.014-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:01:00.014-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:01:00.054-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:01:00.054-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T23:01:00.054-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:01:00.054-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T23:01:00.054-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:01:00.080-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.069 seconds
[2024-04-14T23:02:54.877-0400] {processor.py:161} INFO - Started process (PID=42211) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:02:54.878-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T23:02:54.878-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:02:54.878-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:02:54.916-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:02:54.916-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T23:02:54.916-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:02:54.916-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T23:02:54.916-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:02:54.942-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.066 seconds
[2024-04-14T23:04:54.727-0400] {processor.py:161} INFO - Started process (PID=42328) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:04:54.727-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T23:04:54.728-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:04:54.728-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:04:54.767-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:04:54.767-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T23:04:54.767-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:04:54.767-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T23:04:54.768-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:04:54.793-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.067 seconds
[2024-04-14T23:06:53.979-0400] {processor.py:161} INFO - Started process (PID=42434) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:06:53.981-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T23:06:53.981-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:06:53.981-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:06:54.019-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:06:54.019-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T23:06:54.019-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:06:54.019-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T23:06:54.019-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:06:54.045-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.066 seconds
[2024-04-14T23:08:53.648-0400] {processor.py:161} INFO - Started process (PID=42533) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:08:53.649-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T23:08:53.649-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:08:53.649-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:08:53.688-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:08:53.687-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T23:08:53.688-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:08:53.688-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T23:08:53.688-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:08:53.713-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.067 seconds
[2024-04-14T23:10:43.058-0400] {processor.py:161} INFO - Started process (PID=42621) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:10:43.059-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T23:10:43.059-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:10:43.059-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:10:43.098-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:10:43.097-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T23:10:43.098-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:10:43.098-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T23:10:43.098-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:10:43.123-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.067 seconds
[2024-04-14T23:12:43.766-0400] {processor.py:161} INFO - Started process (PID=42724) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:12:43.767-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T23:12:43.768-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:12:43.768-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:12:43.805-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:12:43.805-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T23:12:43.805-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:12:43.805-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T23:12:43.805-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:12:43.831-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.066 seconds
[2024-04-14T23:14:40.251-0400] {processor.py:161} INFO - Started process (PID=42817) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:14:40.252-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T23:14:40.252-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:14:40.252-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:14:40.290-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:14:40.290-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T23:14:40.290-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:14:40.290-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T23:14:40.290-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:14:40.315-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.065 seconds
[2024-04-14T23:21:52.298-0400] {processor.py:161} INFO - Started process (PID=43003) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:21:52.298-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T23:21:52.299-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:21:52.298-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:21:52.337-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:21:52.337-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T23:21:52.337-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:21:52.337-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T23:21:52.338-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:21:52.363-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.066 seconds
[2024-04-14T23:23:49.784-0400] {processor.py:161} INFO - Started process (PID=43124) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:23:49.786-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T23:23:49.786-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:23:49.786-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:23:49.824-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:23:49.824-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T23:23:49.825-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:23:49.825-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T23:23:49.825-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:23:49.850-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.067 seconds
[2024-04-14T23:25:46.108-0400] {processor.py:161} INFO - Started process (PID=43217) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:25:46.110-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T23:25:46.110-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:25:46.110-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:25:46.147-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:25:46.147-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T23:25:46.147-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:25:46.147-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T23:25:46.147-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:25:46.173-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.065 seconds
[2024-04-14T23:27:41.884-0400] {processor.py:161} INFO - Started process (PID=43315) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:27:41.885-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T23:27:41.885-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:27:41.885-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:27:41.924-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:27:41.924-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T23:27:41.924-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:27:41.924-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T23:27:41.924-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:27:41.950-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.067 seconds
[2024-04-14T23:29:40.724-0400] {processor.py:161} INFO - Started process (PID=43424) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:29:40.724-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T23:29:40.725-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:29:40.724-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:29:40.765-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:29:40.764-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T23:29:40.765-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:29:40.765-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T23:29:40.765-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:29:40.792-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.069 seconds
[2024-04-14T23:31:36.414-0400] {processor.py:161} INFO - Started process (PID=43527) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:31:36.416-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T23:31:36.416-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:31:36.416-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:31:36.454-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:31:36.454-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T23:31:36.454-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:31:36.454-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T23:31:36.454-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:31:36.480-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.066 seconds
[2024-04-14T23:33:35.718-0400] {processor.py:161} INFO - Started process (PID=43625) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:33:35.720-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T23:33:35.720-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:33:35.720-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:33:35.759-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:33:35.759-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T23:33:35.759-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:33:35.759-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T23:33:35.759-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:33:35.785-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.067 seconds
[2024-04-14T23:35:36.182-0400] {processor.py:161} INFO - Started process (PID=43715) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:35:36.183-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T23:35:36.183-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:35:36.183-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:35:36.222-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:35:36.222-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T23:35:36.222-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:35:36.222-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T23:35:36.222-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:35:36.247-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.067 seconds
[2024-04-14T23:37:30.753-0400] {processor.py:161} INFO - Started process (PID=43807) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:37:30.754-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T23:37:30.754-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:37:30.754-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:37:30.806-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:37:30.806-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T23:37:30.806-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:37:30.806-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T23:37:30.806-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:37:30.833-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.083 seconds
[2024-04-14T23:39:20.065-0400] {processor.py:161} INFO - Started process (PID=43898) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:39:20.065-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T23:39:20.066-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:39:20.066-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:39:20.104-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:39:20.104-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T23:39:20.104-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:39:20.104-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T23:39:20.104-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:39:20.130-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.066 seconds
[2024-04-14T23:41:22.873-0400] {processor.py:161} INFO - Started process (PID=43992) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:41:22.875-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T23:41:22.875-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:41:22.875-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:41:22.914-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:41:22.914-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T23:41:22.914-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:41:22.914-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T23:41:22.914-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:41:22.939-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.067 seconds
[2024-04-14T23:43:16.660-0400] {processor.py:161} INFO - Started process (PID=44090) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:43:16.661-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-14T23:43:16.661-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:43:16.661-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:43:16.701-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:43:16.701-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-14T23:43:16.701-0400] {logging_mixin.py:188} INFO - [2024-04-14T23:43:16.701-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-14T23:43:16.701-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-14T23:43:16.727-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.068 seconds
[2024-04-15T00:43:55.901-0400] {processor.py:161} INFO - Started process (PID=44278) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T00:43:55.902-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-15T00:43:55.902-0400] {logging_mixin.py:188} INFO - [2024-04-15T00:43:55.902-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T00:43:55.943-0400] {logging_mixin.py:188} INFO - [2024-04-15T00:43:55.943-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-15T00:43:55.943-0400] {logging_mixin.py:188} INFO - [2024-04-15T00:43:55.943-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-15T00:43:55.943-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T00:43:55.978-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.078 seconds
[2024-04-15T00:45:32.277-0400] {processor.py:161} INFO - Started process (PID=44372) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T00:45:32.278-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-15T00:45:32.278-0400] {logging_mixin.py:188} INFO - [2024-04-15T00:45:32.278-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T00:45:32.326-0400] {logging_mixin.py:188} INFO - [2024-04-15T00:45:32.326-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-15T00:45:32.326-0400] {logging_mixin.py:188} INFO - [2024-04-15T00:45:32.326-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-15T00:45:32.326-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T00:45:32.354-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.078 seconds
[2024-04-15T00:47:39.563-0400] {processor.py:161} INFO - Started process (PID=44487) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T00:47:39.564-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-15T00:47:39.564-0400] {logging_mixin.py:188} INFO - [2024-04-15T00:47:39.564-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T00:47:39.604-0400] {logging_mixin.py:188} INFO - [2024-04-15T00:47:39.604-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-15T00:47:39.604-0400] {logging_mixin.py:188} INFO - [2024-04-15T00:47:39.604-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-15T00:47:39.604-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T00:47:39.629-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.067 seconds
[2024-04-15T00:49:38.864-0400] {processor.py:161} INFO - Started process (PID=44590) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T00:49:38.864-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-15T00:49:38.865-0400] {logging_mixin.py:188} INFO - [2024-04-15T00:49:38.864-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T00:49:38.903-0400] {logging_mixin.py:188} INFO - [2024-04-15T00:49:38.903-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-15T00:49:38.903-0400] {logging_mixin.py:188} INFO - [2024-04-15T00:49:38.903-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-15T00:49:38.903-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T00:49:38.929-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.066 seconds
[2024-04-15T00:51:38.371-0400] {processor.py:161} INFO - Started process (PID=44677) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T00:51:38.373-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-15T00:51:38.373-0400] {logging_mixin.py:188} INFO - [2024-04-15T00:51:38.373-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T00:51:38.412-0400] {logging_mixin.py:188} INFO - [2024-04-15T00:51:38.412-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-15T00:51:38.412-0400] {logging_mixin.py:188} INFO - [2024-04-15T00:51:38.412-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-15T00:51:38.412-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T00:51:38.439-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.069 seconds
[2024-04-15T00:53:38.600-0400] {processor.py:161} INFO - Started process (PID=44769) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T00:53:38.600-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-15T00:53:38.601-0400] {logging_mixin.py:188} INFO - [2024-04-15T00:53:38.601-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T00:53:38.640-0400] {logging_mixin.py:188} INFO - [2024-04-15T00:53:38.639-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-15T00:53:38.640-0400] {logging_mixin.py:188} INFO - [2024-04-15T00:53:38.640-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-15T00:53:38.640-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T00:53:38.667-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.068 seconds
[2024-04-15T00:55:38.943-0400] {processor.py:161} INFO - Started process (PID=44865) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T00:55:38.944-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-15T00:55:38.945-0400] {logging_mixin.py:188} INFO - [2024-04-15T00:55:38.945-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T00:55:38.983-0400] {logging_mixin.py:188} INFO - [2024-04-15T00:55:38.983-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-15T00:55:38.983-0400] {logging_mixin.py:188} INFO - [2024-04-15T00:55:38.983-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-15T00:55:38.983-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T00:55:39.009-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.066 seconds
[2024-04-15T00:57:37.635-0400] {processor.py:161} INFO - Started process (PID=44955) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T00:57:37.636-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-15T00:57:37.636-0400] {logging_mixin.py:188} INFO - [2024-04-15T00:57:37.636-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T00:57:37.675-0400] {logging_mixin.py:188} INFO - [2024-04-15T00:57:37.675-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-15T00:57:37.676-0400] {logging_mixin.py:188} INFO - [2024-04-15T00:57:37.676-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-15T00:57:37.676-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T00:57:37.701-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.067 seconds
[2024-04-15T03:41:22.472-0400] {processor.py:161} INFO - Started process (PID=45256) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T03:41:22.473-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-15T03:41:22.473-0400] {logging_mixin.py:188} INFO - [2024-04-15T03:41:22.473-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T03:41:22.511-0400] {logging_mixin.py:188} INFO - [2024-04-15T03:41:22.511-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-15T03:41:22.512-0400] {logging_mixin.py:188} INFO - [2024-04-15T03:41:22.512-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-15T03:41:22.512-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T03:41:22.537-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.066 seconds
[2024-04-15T06:54:47.943-0400] {processor.py:161} INFO - Started process (PID=45461) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T06:54:47.944-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-15T06:54:47.944-0400] {logging_mixin.py:188} INFO - [2024-04-15T06:54:47.944-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T06:54:47.983-0400] {logging_mixin.py:188} INFO - [2024-04-15T06:54:47.983-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-15T06:54:47.983-0400] {logging_mixin.py:188} INFO - [2024-04-15T06:54:47.983-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-15T06:54:47.983-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T06:54:48.009-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.066 seconds
[2024-04-15T07:06:44.654-0400] {processor.py:161} INFO - Started process (PID=45564) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T07:06:44.655-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-15T07:06:44.655-0400] {logging_mixin.py:188} INFO - [2024-04-15T07:06:44.655-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T07:06:44.701-0400] {logging_mixin.py:188} INFO - [2024-04-15T07:06:44.701-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-15T07:06:44.701-0400] {logging_mixin.py:188} INFO - [2024-04-15T07:06:44.701-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-15T07:06:44.701-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T07:06:44.730-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.078 seconds
[2024-04-15T07:34:34.312-0400] {processor.py:161} INFO - Started process (PID=45660) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T07:34:34.313-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-15T07:34:34.313-0400] {logging_mixin.py:188} INFO - [2024-04-15T07:34:34.313-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T07:34:34.351-0400] {logging_mixin.py:188} INFO - [2024-04-15T07:34:34.351-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-15T07:34:34.351-0400] {logging_mixin.py:188} INFO - [2024-04-15T07:34:34.351-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-15T07:34:34.351-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T07:34:34.378-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.066 seconds
[2024-04-15T10:41:05.134-0400] {processor.py:161} INFO - Started process (PID=45850) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T10:41:05.135-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-15T10:41:05.135-0400] {logging_mixin.py:188} INFO - [2024-04-15T10:41:05.135-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T10:41:05.178-0400] {logging_mixin.py:188} INFO - [2024-04-15T10:41:05.177-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-15T10:41:05.178-0400] {logging_mixin.py:188} INFO - [2024-04-15T10:41:05.178-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-15T10:41:05.178-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T10:41:05.205-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.072 seconds
[2024-04-15T12:49:34.686-0400] {processor.py:161} INFO - Started process (PID=46005) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T12:49:34.687-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-15T12:49:34.687-0400] {logging_mixin.py:188} INFO - [2024-04-15T12:49:34.687-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T12:49:34.725-0400] {logging_mixin.py:188} INFO - [2024-04-15T12:49:34.725-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-15T12:49:34.725-0400] {logging_mixin.py:188} INFO - [2024-04-15T12:49:34.725-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-15T12:49:34.725-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T12:49:34.751-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.066 seconds
[2024-04-15T12:53:33.506-0400] {processor.py:161} INFO - Started process (PID=46104) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T12:53:33.507-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-15T12:53:33.508-0400] {logging_mixin.py:188} INFO - [2024-04-15T12:53:33.507-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T12:53:33.547-0400] {logging_mixin.py:188} INFO - [2024-04-15T12:53:33.546-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-15T12:53:33.547-0400] {logging_mixin.py:188} INFO - [2024-04-15T12:53:33.547-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-15T12:53:33.547-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T12:53:33.573-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.068 seconds
[2024-04-15T12:55:39.500-0400] {processor.py:161} INFO - Started process (PID=46208) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T12:55:39.501-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-15T12:55:39.501-0400] {logging_mixin.py:188} INFO - [2024-04-15T12:55:39.501-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T12:55:39.539-0400] {logging_mixin.py:188} INFO - [2024-04-15T12:55:39.539-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-15T12:55:39.539-0400] {logging_mixin.py:188} INFO - [2024-04-15T12:55:39.539-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-15T12:55:39.540-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T12:55:39.565-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.066 seconds
[2024-04-15T12:57:34.869-0400] {processor.py:161} INFO - Started process (PID=46304) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T12:57:34.870-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-15T12:57:34.870-0400] {logging_mixin.py:188} INFO - [2024-04-15T12:57:34.870-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T12:57:34.909-0400] {logging_mixin.py:188} INFO - [2024-04-15T12:57:34.909-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-15T12:57:34.909-0400] {logging_mixin.py:188} INFO - [2024-04-15T12:57:34.909-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-15T12:57:34.909-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T12:57:34.935-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.067 seconds
[2024-04-15T13:10:16.023-0400] {processor.py:161} INFO - Started process (PID=46418) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T13:10:16.024-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-15T13:10:16.024-0400] {logging_mixin.py:188} INFO - [2024-04-15T13:10:16.024-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T13:10:16.063-0400] {logging_mixin.py:188} INFO - [2024-04-15T13:10:16.063-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-15T13:10:16.063-0400] {logging_mixin.py:188} INFO - [2024-04-15T13:10:16.063-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-15T13:10:16.063-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T13:10:16.089-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.067 seconds
[2024-04-15T13:12:19.864-0400] {processor.py:161} INFO - Started process (PID=46518) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T13:12:19.865-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-15T13:12:19.865-0400] {logging_mixin.py:188} INFO - [2024-04-15T13:12:19.865-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T13:12:19.904-0400] {logging_mixin.py:188} INFO - [2024-04-15T13:12:19.903-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-15T13:12:19.904-0400] {logging_mixin.py:188} INFO - [2024-04-15T13:12:19.904-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-15T13:12:19.904-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T13:12:19.929-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.066 seconds
[2024-04-15T13:14:22.491-0400] {processor.py:161} INFO - Started process (PID=46615) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T13:14:22.492-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-15T13:14:22.492-0400] {logging_mixin.py:188} INFO - [2024-04-15T13:14:22.492-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T13:14:22.534-0400] {logging_mixin.py:188} INFO - [2024-04-15T13:14:22.534-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-15T13:14:22.534-0400] {logging_mixin.py:188} INFO - [2024-04-15T13:14:22.534-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-15T13:14:22.534-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T13:14:22.560-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.070 seconds
[2024-04-15T13:16:28.332-0400] {processor.py:161} INFO - Started process (PID=46723) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T13:16:28.333-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-15T13:16:28.333-0400] {logging_mixin.py:188} INFO - [2024-04-15T13:16:28.333-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T13:16:28.375-0400] {logging_mixin.py:188} INFO - [2024-04-15T13:16:28.375-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-15T13:16:28.375-0400] {logging_mixin.py:188} INFO - [2024-04-15T13:16:28.375-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-15T13:16:28.375-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T13:16:28.401-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.070 seconds
[2024-04-15T13:18:44.390-0400] {processor.py:161} INFO - Started process (PID=46864) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T13:18:44.391-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-15T13:18:44.391-0400] {logging_mixin.py:188} INFO - [2024-04-15T13:18:44.391-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T13:18:44.435-0400] {logging_mixin.py:188} INFO - [2024-04-15T13:18:44.435-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-15T13:18:44.435-0400] {logging_mixin.py:188} INFO - [2024-04-15T13:18:44.435-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-15T13:18:44.435-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T13:18:44.461-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.072 seconds
[2024-04-15T13:20:14.872-0400] {processor.py:161} INFO - Started process (PID=46943) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T13:20:14.876-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-15T13:20:14.877-0400] {logging_mixin.py:188} INFO - [2024-04-15T13:20:14.877-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T13:20:14.953-0400] {logging_mixin.py:188} INFO - [2024-04-15T13:20:14.953-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-15T13:20:14.953-0400] {logging_mixin.py:188} INFO - [2024-04-15T13:20:14.953-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-15T13:20:14.953-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T13:20:14.982-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.116 seconds
[2024-04-15T13:22:26.085-0400] {processor.py:161} INFO - Started process (PID=47056) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T13:22:26.086-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-15T13:22:26.086-0400] {logging_mixin.py:188} INFO - [2024-04-15T13:22:26.086-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T13:22:26.126-0400] {logging_mixin.py:188} INFO - [2024-04-15T13:22:26.126-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-15T13:22:26.127-0400] {logging_mixin.py:188} INFO - [2024-04-15T13:22:26.127-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-15T13:22:26.127-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T13:22:26.153-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.069 seconds
[2024-04-15T13:24:46.003-0400] {processor.py:161} INFO - Started process (PID=47198) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T13:24:46.004-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-15T13:24:46.004-0400] {logging_mixin.py:188} INFO - [2024-04-15T13:24:46.004-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T13:24:46.050-0400] {logging_mixin.py:188} INFO - [2024-04-15T13:24:46.050-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-15T13:24:46.050-0400] {logging_mixin.py:188} INFO - [2024-04-15T13:24:46.050-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-15T13:24:46.050-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T13:24:46.077-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.076 seconds
[2024-04-15T13:27:14.958-0400] {processor.py:161} INFO - Started process (PID=47337) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T13:27:14.959-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-15T13:27:14.959-0400] {logging_mixin.py:188} INFO - [2024-04-15T13:27:14.959-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T13:27:15.032-0400] {logging_mixin.py:188} INFO - [2024-04-15T13:27:15.032-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-15T13:27:15.032-0400] {logging_mixin.py:188} INFO - [2024-04-15T13:27:15.032-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-15T13:27:15.033-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T13:27:15.076-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.124 seconds
[2024-04-15T13:29:42.697-0400] {processor.py:161} INFO - Started process (PID=47476) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T13:29:42.697-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-15T13:29:42.698-0400] {logging_mixin.py:188} INFO - [2024-04-15T13:29:42.698-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T13:29:42.741-0400] {logging_mixin.py:188} INFO - [2024-04-15T13:29:42.741-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-15T13:29:42.741-0400] {logging_mixin.py:188} INFO - [2024-04-15T13:29:42.741-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-15T13:29:42.742-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T13:29:42.767-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.072 seconds
[2024-04-15T13:31:59.824-0400] {processor.py:161} INFO - Started process (PID=47624) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T13:31:59.824-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-15T13:31:59.825-0400] {logging_mixin.py:188} INFO - [2024-04-15T13:31:59.825-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T13:31:59.864-0400] {logging_mixin.py:188} INFO - [2024-04-15T13:31:59.863-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-15T13:31:59.864-0400] {logging_mixin.py:188} INFO - [2024-04-15T13:31:59.864-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-15T13:31:59.864-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T13:31:59.890-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.067 seconds
[2024-04-15T13:35:33.410-0400] {processor.py:161} INFO - Started process (PID=47880) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T13:35:33.411-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-15T13:35:33.411-0400] {logging_mixin.py:188} INFO - [2024-04-15T13:35:33.411-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T13:35:33.454-0400] {logging_mixin.py:188} INFO - [2024-04-15T13:35:33.453-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-15T13:35:33.454-0400] {logging_mixin.py:188} INFO - [2024-04-15T13:35:33.454-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-15T13:35:33.454-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T13:35:33.482-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.073 seconds
[2024-04-15T19:10:29.231-0400] {processor.py:161} INFO - Started process (PID=49695) to work on /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T19:10:29.233-0400] {processor.py:830} INFO - Processing file /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-04-15T19:10:29.233-0400] {logging_mixin.py:188} INFO - [2024-04-15T19:10:29.233-0400] {dagbag.py:540} INFO - Filling up the DagBag from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T19:10:29.297-0400] {logging_mixin.py:188} INFO - [2024-04-15T19:10:29.296-0400] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-04-15T19:10:29.297-0400] {logging_mixin.py:188} INFO - [2024-04-15T19:10:29.297-0400] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-04-15T19:10:29.297-0400] {processor.py:842} WARNING - No viable dags retrieved from /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-04-15T19:10:29.345-0400] {processor.py:183} INFO - Processing /Users/manideepakreddyaila/Desktop/projects/Legal_Insights_Engine/.venv/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.116 seconds
